modules/routes/variables.tf

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  description = "The ID of the project where the routes will be created"
+}
+
+variable "network_name" {
+  description = "The name of the network where routes will be created"
+}
+
+variable "network" {
```

Comment:
Isn't this redundant with `network_name`? Don't think we should need both.

Reply:
This just followed the original, single module, functionality. `network` and `subnet` resources were added to the `depends_on` black of the routes to force the depedency. 

But passing in just the `network_name` from the output of the vpc submodule should force that dependency anyways. Ill remove `network`